### PR TITLE
added note about sources of truth

### PR DESCRIPTION
**Pull request by mportman12**
Added note to the About section to clarify which guide is the source of truth for what type of information. Based on [Issue#74](https://github.com/redhat-documentation/supplementary-style-guide/issues/74).

### CODE REVIEW

**Comment by mportman12** [comment]
@stoobie please review

File: supplementary_style_guide/introduction/about-guide.adoc
Comment (stoobie) [inline supplementary_style_guide/introduction/about-guide.adoc:11]: @mportman12 Thank you so much for creating this PR! The text looks good to me. But I would move this from being a note to a full fledged section with a subheading under How to use this guide, or even as a heading 2 either before or after How to use this guide.

Comment (mportman12) [reply supplementary_style_guide/introduction/about-guide.adoc]: @stoobie Moved the paragraph to a new section. I also removed the link to the SSG because it's irrelevant - the reader is already in the SSG.

File: supplementary_style_guide/introduction/about-guide.adoc
Comment (stoobie) [inline supplementary_style_guide/introduction/about-guide.adoc:11]: @mportman12 This is kind of wishy-washy. If the point of this PR is to provide the guidance on the source of truth, then don't be shy about saying that here. This is the guidance we want to provide here, so I think the heading should be:
`== Source of truth`

@ndeevy @bergerhoffer @cbudz @inoxx03 Curious to know what others think.

Comment (bergerhoffer) [reply supplementary_style_guide/introduction/about-guide.adoc:11]: I understand that we want to clarify the source of truth, but I don't think that we necessarily need to use that phrasing.

As for the existing wording, I wouldn't necessarily call them style guides, because that might get confusing (because the SSG is the style guide). 

"Related Red Hat product documentation guides" or "Official Red Hat ..." or something along that lines.

**Review by ndeevy — DISMISSED** [review]

File: supplementary_style_guide/introduction/about-guide.adoc
Comment (stoobie) [reply supplementary_style_guide/introduction/about-guide.adoc:3]: @mportman12 I understand, but this guide is publicly facing. It's one thing to mention that we use the IBM Style Guide, but I don't feel good providing a link that results in this message.

But I'm wishy-washy. @inoxx03 @bergerhoffer WDYT?

Comment (ndeevy) [reply supplementary_style_guide/introduction/about-guide.adoc:3]: 2 cents:
I'd lean towards removing the IBM link for the reasons Steve mentions. Throughout this guide we mention the _IBM Style Guide_  and here could be no different. If we just say its name then there's no need to worry about who can or can't access (like upstream folks) the guide at this link.

Comment (bergerhoffer) [reply supplementary_style_guide/introduction/about-guide.adoc:3]: Yeah I don't think providing the link is necessary. Some might use this site to access the ISG, others might have a downloaded PDF. I think we can leave it up to the reader on how they get their ISG copy.

Comment (mportman12) [reply supplementary_style_guide/introduction/about-guide.adoc]: Removed ISG link.

**Review by bergerhoffer — APPROVED** [review]

**Review by stoobie — APPROVED** [review]
LGTM!

**Comment by bergerhoffer** [comment]
Counting @ndeevy's LGTM before the ISG link was removed is 3 approvals. Merging!

**Pull request merged**
